lib/git/status.rb

```diff
@@ -9,19 +9,35 @@ def initialize(base)
     end
     
     def changed
-      @files.select { |k, f| f.type == 'M' }
+      @files.select { |_k, f| f.type == 'M' }
+    end
+
+    def changed?(file)
+      changed.member?(file)
     end
     
     def added
-      @files.select { |k, f| f.type == 'A' }
+      @files.select { |_k, f| f.type == 'A' }
+    end
+
+    def added?(file)
+      added.member?(file)
     end
 
     def deleted
-      @files.select { |k, f| f.type == 'D' }
+      @files.select { |_k, f| f.type == 'D' }
+    end
+
+    def deleted?(file)
+      deleted.member?(file)
     end
     
     def untracked
-      @files.select { |k, f| f.untracked }
+      @files.select { |_k, f| f.untracked }
+    end
+
+    def untracked?(file)
+      untracked.member?(file)
```

Comment:
For the new methods added, would you mind adding YARD-style comments that describes briefly:

- What they do
- What the expected types of the input and return values are

The new methods added in this PR are:

- changed?
- added?
- deleted?
- untracked?

Something like this will do:

```ruby
#
# Determines whether the given file has been added to the repository
#
# @param file [String] The name of the file.
# @return [Boolean]
```

This way, it is clearer for readers of the code what the intent of these (public API methods) is.